File: src/main/java/ar/edu/um/tesoreria/rest/extern/model/BajaFacultad.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.extern.model;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.OffsetDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class BajaFacultad implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 8260580906592802175L;

	private Integer facultadId;
	private BigDecimal personaId;
	private Integer documentoId;
	private Integer lectivoId;
	private Long bajaId;
	private Byte definitiva;
	private Byte certificadoanaliticoparcial;
	private Byte libredeuda;
	private Byte previa;
	private Byte temporaria;
	private String motivo;
	private OffsetDateTime fecha;
	
}

File: src/main/java/ar/edu/um/tesoreria/rest/extern/model/kotlin/BajaFacultad.kt
package ar.edu.um.tesoreria.rest.extern.model.kotlin

import java.math.BigDecimal
import java.time.OffsetDateTime

data class BajaFacultad(

    var facultadId: Int? = null,
    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var lectivoId: Int? = null,
    var bajaId: Long? = null,
    var definitiva: Byte = 0,
    var certificadoanaliticoparcial: Byte = 0,
    var libredeuda: Byte = 0,
    var previa: Byte = 0,
    var temporaria: Byte = 0,
    var motivo: String? = null,
    var fecha: OffsetDateTime? = null,

    )
